apis/v1alpha1/neuronmonitor_types.go

```diff
@@ -62,6 +62,10 @@ type NeuronMonitorSpec struct {
 	// consumed in the config file for the Collector.
 	// +optional
 	Env []v1.EnvVar `json:"env,omitempty"`
+	// Toleration to schedule OpenTelemetry Collector pods.
```

Comment:
Same here.. this should say Neuron Monitor pods

Reply:
Will fix.